elemental-system-agent/elemental-system-agent.changes

```diff
@@ -1,3 +1,22 @@
+-------------------------------------------------------------------
+Mon Sep 22 11:20:03 UTC 2025 - David Cassany <dcassany@suse.com>
+
+- Update to version 0.3.13:
+  * Bumped dependencies for k8s v1.33
+  * Add delete for plan.File
+  * fix dispatch
+  * fix: add retry logic for one time instruction
+  * Get UID/GID for current user in write file_test.go
+  * Update secrets for dispatch
+  * fix golangci
+  * support k8s 1.32.2
```

Comment:
I think that newer versions will work as well? As we already have v1.37.x in `rancher/elemental` CI.